gluster-utilization.libsonnet

```diff
@@ -0,0 +1,66 @@
+{
+  prometheusAlerts+:: {
+    groups+: [
+      {
+        name: 'gluster-utilization',
+        rules: [
+          {
+            alert: 'GlusterVolumeUtilization',
+            expr: |||
+                  100 * gluster:volume_capacity_used_bytes_total:sum > 80,
+            |||,
+            'for': '5m',
```

Comment:
How to make this time `5m` configurable?

Reply:
@shtripat  @cloudbehl  made it configurable.